pkg/sql/opt/ordering/ordering.go

```diff
@@ -31,6 +31,30 @@ func CanProvide(expr memo.RelExpr, required *props.OrderingChoice) bool {
 	return funcMap[expr.Op()].canProvideOrdering(expr, required)
 }
 
+// CanEnforce returns true if the output of the given operator can be sorted
+// in order to satisfy the given required ordering.
+func CanEnforce(expr memo.RelExpr, required *props.OrderingChoice) bool {
+	if required.Any() {
+		return false
+	}
+	if buildutil.CrdbTestBuild {
+		checkRequired(expr, required)
+	}
+	switch t := expr.(type) {
+	case *memo.ExplainExpr:
+		return false
+	case *memo.LookupJoinExpr:
+		// For paired joins we use a boolean continuation column that to handle
+		// false positive matches in the first join. This relies on the ordering
+		// being unchanged between the first and second joins, so enforcing a
+		// sort on top of this expression could lead to incorrect results.
```

Comment:
nit: that to handle -> to handle
nit: enforcing a sort -> adding a sort (or enforcing an ordering)

Reply:
Done.